lib/ws-server.js

```diff
@@ -0,0 +1,68 @@
+module.exports = WebSocketServer;
+
+var assert = require('assert');
+var debug = require('./debug')('process:' + process.pid);
+var message = require('./message');
+var util = require('util');
+var url = require('url');
+var Server = require('ws').Server;
+var WebSocketChannel = require('./ws-channel');
+
+
+function WebSocketServer(options, callback) {
+  if (typeof options === 'string') {
+    options = url.parse(options);
+    options.host = options.hostname;
+  }
+
+  Server.call(this, options, callback);
+
+  this._channels = {};
+  this.on('connection', this._onConnection.bind(this));
+}
+
+util.inherits(WebSocketServer, Server);
+
+
+WebSocketServer.prototype.unref = function unref() {
+  return this._server.unref();
+};
+
+
+WebSocketServer.prototype.createChannel =
+    function createChannel(onRequest, onNotification) {
```

Comment:
No reason, I just did it (for public methods) because the code that was already there did it too.